d3a_api_client/utils.py

```diff
@@ -369,3 +370,12 @@ def read_simulation_config_file(config_file_path):
 def get_sim_id_and_domain_names():
     return simulation_id_from_env(), domain_name_from_env(), websocket_domain_name_from_env()
 
+
+def validate_client_up_to_date(response):
+    if "Api-Version" in response.headers:
+        remote_version = response.headers.get("Api-Version")
```

Comment:
When I see the corresponding d3a-web PR, the api version is saved in the `API-VERSION` member of the header, not in `Api-Version`. Can you please explain?

Reply:
![Screen Shot 2021-04-30 at 10 16 56](https://user-images.githubusercontent.com/38125715/116661511-37705380-a99d-11eb-91cf-98e2ded73d17.png)
@hannesdiedrich It seems all response headers are returned in this case, whatever the sent key was AuThOrIzatiON-HEADER is translated to Authorization-header, I'm not 100% sure which party is responsible for this behavior though

Reply:
Darn, I could not find anything in the internet ( or rather I could not find appropriate search words :-) 
@fievelk Do you have an idea why `request` or something else changes these keys here?

Reply:
HTTP headers' keys are case-insensitive (see: [RFC-7230](https://tools.ietf.org/html/rfc7230) and [this SO question](https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive)).
I wouldn't be too surprised if `requests` would allow you to query the headers with both `response.headers["Api-Version"]` and `response.headers["API_VERSION"]`. It would be interesting if you could give it a try.

Reply:
Wow, thanks! I suggest to go with a consistent `"API-VERSION"` ok?

Reply:
In order to complete the picture here, the CamelCase is just how Chrome developer tools present the headers. The protocol itself dictates the headers to be lower case before encoded https://tools.ietf.org/html/rfc7540#section-8.1.2 . Therefore all HTTP-compliant clients should be able to retrieve header values no matter the style of the header. To the actual comment I personally have no strong opinion on that, but uppercase looks more familiar to me due to habit.